docs/framework/react/guides/custom-errors.md

```diff
@@ -0,0 +1,267 @@
+---
+id: custom-errors
+title: Custom Errors
+---
+
+
+
+
+## Return String Values from Forms
+
+By default, TanStack Form allows you to return string error messages from validators:
```

Comment:
I would rephrase this one since strings are (no longer) the default error in TanStack Form, the user decides what the standard should be.

You can return literally anything from the validators, but to make the explanation simple it totally makes sense to begin with strings.